src/index.tsx

```diff
@@ -30,9 +29,6 @@ if (process.env.REACT_APP_NODE_ENV === 'development' && (window as any).ethereum
   mockLocalhostWeb3Provider(window);
 }
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+const root = createRoot(document.getElementById('root')!);
+
+root.render(<App />);
```

Comment:
I support your decision about not using strict mode, but add a comment about that. We will forget about this after some time and wonder.

Reply:
Good idea 👍